onnxruntime/contrib_ops/cuda/transformers/generation_device_helper.cc

```diff
@@ -445,6 +471,44 @@ Status ProcessLogits(const OrtValue& logits,                                 //
       parameters->no_repeat_ngram_size,
       cuda_stream);
 
+  // Whisper time stamp generation.
+  // TODO: implement it on GPU
+  bool gen_timestamp = is_whisper_model &&
+                       (parameters->logits_processor == onnxruntime::contrib::transformers::IGenerationParameters::kLogitsProcessorTypeWhisper);
+  if (gen_timestamp) {
+    ; // Logic largely affected by beamsearch changes, change them later
```

Comment:
what's plan to deal this part?

Reply:
done! make current work, will make GPU version next step.